### ISSUE TITLE
AirPlay Support

### ISSUE THREAD

**Issue by jaruba** [jaruba/PowderPlayer#37]

**Comment by Mel79** [comment]
Will there be AirPlay support added. This would be very useful now that the TorrenTV app no longer works.

**Comment by jaruba** [comment]
I'd do it but we have no testing environment for AirPlay. Are you up to testing it for us when the time comes?

**Comment by Mel79** [comment]
Yes, I’d be happy to test it for you.

Mel Blackburn

**Comment by Mel79** [comment]
I think that AirPlay is the only thing missing right now. PowderPlayer with AirPlay is an excellent idea. It would be like combining 5KPlayer and TorrenTV into one app.

**Comment by jaruba** [comment]
Powder Player Nightly Pre-Release now supports Airplay with subtitles and advanced subtitle features. Chromecast and Advanced DLNA support is also available.

And as you mentioned 5KPlayer, I also added the YouTube Downloader library, so it now supports all these sites too:
https://rg3.github.io/youtube-dl/supportedsites.html

The official announcement for Powder Player Nightly Pre-Release and the download link can be found here:
https://www.reddit.com/r/PowderPlayer/comments/4ralk2/powder_player_v100_prerelease_available_for/

I tested it on an Apple TV 3 and an Apple TV 4 and worked gloriously.
